main.js

```diff
@@ -1 +1,109 @@
-console.log('CONNECTED')
+// ** Data Types **//
+const students = [
+
+];
+
+// ** DOM Printer  **//
+const printToDom = (divId, textToPrint) => {
+  const selectedDiv = document.querySelector(divId);
+  selectedDiv.innerHTML = textToPrint;
+};
+
+// ** HTML Builders **//
+const studentBuilder = (taco) => {
+  let domString = ' ';
+
+  for (const student of taco) {
+    domString += `<div class="card m-2" style="width: 18rem;">
+    <div class="card-body">
+      <h5 class="card-title">${student.name}</h5>
+      <p class="card-text">${student.house}</p>
+      <button type="button" class="btn btn-secondary" id="${student.id}">Expel</button>
+      </div> 
+    </div>`;
+  }
+  printToDom('#studentDiv', domString)
+}
+
+const warningMessage = () => {
+  messageString = '<h3>Please type in a name</h3>'
+  printToDom('#warningMessage', messageString)
+};
+//** Event Handlers **//
+
+const newStudent = (e) => {
+  e.preventDefault();
+
+  const availableHouses = ['Gryffindor', 'Slytherin', 'Ravenclaw', 'Huffelpuff']
+
+  const housePicker = availableHouses[Math.floor(Math.random() * availableHouses.length)]
+
+  // Target form values //
+  const studentName = document.querySelector('#inputStudentName').value;
+  // Create unique ID for students 
+  const studentIds = students.map(student => student.id).sort((a,b) => a - b);
+
+  const id = studentIds.length ? studentIds[(studentIds.length - 1)] + 1 : 1;
+  // Store values in obj //
+  const obj = {
```

Comment:
@caseywalker To make your code a little cleaner you can use Object shorthand notation. Maybe change your variable names to: `name`, `house`, and `id` then your object can look like this:
```
const obj = {
  name,
  house,
  id
}
```

Reply:
@drteresavasquez Loved the feedback and suggestion. I have cleaned up my function and am now utilizing object shorthand. The code is much easier to read! 

https://github.com/caseywalker/sorting-hat/pull/27